serverlessworkflow/modules/ROOT/pages/use-cases/advanced-developer-use-cases/integrations/custom-functions-python.adoc

```diff
@@ -0,0 +1,116 @@
+= Invoking Python from {product_name}
+:compat-mode!:
+// Metadata:
+:description: Describe Python execution capabilities
+:keywords: kogito, workflow, quarkus, serverless, python, AI
+
+This document describes how to integrate python scripts and function into you workflow using {product_name} custom functions. The code appearing in this document is copied from link:{kogito_sw_examples_url}/serverless-workflow-python-quarkus[`serverless-workflow-python-quarkus`] example application and link:{kogito_runtimes_url}/quarkus/addons/python/integration-tests/src/main/resources/PythonService.sw.json[PythonService] integration test.
+
+== Enable Python support
+
+ To enable Python support you need to the python add-on dependency to your {prouct_name} module `pom.xml` file
+
+[source,xml]
+----
+<dependency>
+      <groupId>org.apache.kie.sonataflow</groupId>
+      <artifactId>sonataflow-addons-quarkus-python</artifactId>
+</dependency>
+----
+
+== Invoking embedded Python script.
+
+{product_name} supports of execution python script in the same memory address than the running workflow.
+
+To invoke a python script the first step is to define a custom python function at the beginning of the flow. 
+
+[source,json]
+----
+ "functions": [
+    {
+      "name": "python",
+      "type": "custom",
+      "operation": "script:python"
+    }
+  ]
+----
+
+Once done, you can use that function several times to execute arbitrary python code. The python code is provided as argument of the function call through `script` property. 
+
+[source,json]
+----
+"functionRef": 
+   "name" : "Imports",
```

Comment:
Shouldn't it be lowercase following python function naming convention?

Reply:
Actually this name should not be there at all ;)